tests/unit/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_executable(runInterpolationTests test_unit_interpolators_g.cpp)
+target_link_libraries(runInterpolationTests Grackle_Grackle gtest gtest_main hdf5)
+target_include_directories(runInterpolationTests PRIVATE ${PROJECT_SOURCE_DIR}/src/clib)
```

Comment:
It might look like I'm forgetting dependencies here (i.e. ``gtest`` and ``hdf5``), but these are actually transitive dependencies of the other targets. So they are automatically included.

Reply:
Done. BTW, I had the same DLINUX issue on PizDaint.